proof/refine/ARM/IpcCancel_R.thy

```diff
@@ -934,8 +934,8 @@ lemma (in delete_one_conc_pre) cancelIPC_st_tcb_at:
    \<lbrace>\<lambda>rv. st_tcb_at' P t\<rbrace>"
   apply (simp add: cancelIPC_def Let_def getThreadReplySlot_def
              cong: if_cong Structures_H.thread_state.case_cong)
-  apply (rule hoare_seq_ext [OF _ gts_sp'])
-  apply (case_tac x, simp_all add: isTS_defs list_case_If)
+  apply (rule bind_wp [OF _ gts_sp'])
+  apply (case_tac rv, simp_all add: isTS_defs list_case_If)
```

Comment:
This is an example of the third category of proof changes from the PR description; the `bind_wp` step now names the bound variable produced `rv`.